src/actions/responseHandler.js

```diff
@@ -0,0 +1,33 @@
+import { toast } from 'react-toastify';
+
+
+const responseHandler = {
+
+  updateBoard(data, symbol, setSymbol, setBoard) {
+    if(data.board) setBoard(data.board)
+    responseHandler.handleResponse(data, symbol, setSymbol)
+  },
+
+  handleResponse(data, symbol, setSymbol) {
```

Comment:
As discussed in the session, it may be better to encapsulate switching symbols in a `success` callback and error handling in an `error` callback so that response does not need to know the details of how the game works. It would just invoke `success` when `data.message` is Ok, and `error` otherwise.